README-PDF.md

```diff
@@ -0,0 +1,52 @@
+# Advanced-Reports: PDF Reports
+
+PDF reports generate a PDF file from a *Source Report* (which provides the data to include in the
+report) and the *HTML Source* (which provides the document template/layout).
+
+## Source Report
+
+The source report for a PDF report can be any Instrument Query.
+
+## Paper Size
+
+Choose the paper size and orientation for your PDF file.
+
+## HTML Source
+
+Create your document template using HTML. The module will convert this to a PDF.
+
+You can use *field variables* and *logic tags* to include the data from the source report into your
+PDF report.
+
+**Field Variables** consist of the name of the field or column as output by the source report,
```

Comment:
I think this needs to make clear that this is alias name if used

Reply:
Amended.